external/multi-base-images/spark/buildLightbendSpark.sh

```diff
@@ -45,11 +47,18 @@ $DIR/spark/dev/make-distribution.sh --name cloudflow-2.12 --tgz -Pscala-2.12 -Ph
 
 # build the Spark image
 tar -zxvf spark-$SPARK_VERSION-bin-cloudflow-2.12.tgz
-cd $DIR/spark/spark-$SPARK_VERSION-bin-cloudflow-2.12
+# replace protobuf jar to support sparksql-scalapb and load Scala-PB
```

Comment:
Do you have an idea for getting this to work without needing our own build of Spark?

Reply:
Not really, unless Spark will update their protobuf version.
At any rate we are using out own image so that we can inject spark-entrypoint.sh
Its not really a different build its just replacing protobufs jar and adding scalapb jar